.devops/nix/package.nix

```diff
@@ -198,6 +201,9 @@ effectiveStdenv.mkDerivation (
         (cmakeBool "LLAMA_VULKAN" useVulkan)
         (cmakeBool "LLAMA_STATIC" enableStatic)
       ]
+      ++ optionals crossCompileWindows [
+        (cmakeFeature "CMAKE_SYSTEM_NAME" "Windows")
+      ]
```

Comment:
Is https://github.com/NixOS/nixpkgs/blob/13a8da967119594974797df76e0fbf43194303b1/pkgs/build-support/lib/cmake.nix#L8-L9 not enough?

Reply:
Oh yes, you are completely correct; this is not needed anymore after I've just started overwriting stdenv. Well spottet, thank you!